src/abstractdataframe/selection.jl

```diff
@@ -392,7 +392,15 @@ function _transformation_helper(df::AbstractDataFrame, col_idx::AbstractVector{I
         return _empty_selector_helper(fun.fun, nrow(df))
     else
         cdf = eachcol(df)
-        return fun(map(c -> cdf[c], col_idx)...)
+        cols = map(c -> cdf[c], col_idx)
+        if (fun === +) || fun === ByRow(+)
+            return _sum_fast(cols)
+        elseif fun === ByRow(min)
+            return _minmax_row_fast(cols, min)
+        elseif fun === ByRow(max)
+            return _minmax_row_fast(cols, max)
+        end
+        return fun(cols...)
```

Comment:
Shouldn't other reductions be checked here too?

Also:
```suggestion
        else
            return fun(cols...)
        end
```

Reply:
What other reductions do you have in mind? Only `+`, `min` and `max` make sense for positional arguments. I will add `*` when I implement it.

Reply:
Actually given your comment for `Base.add_sum` above, maybe we should drop the fast path for positional arguments altogether? Given e.g.:
```
julia> sum([0x1, 0x1])
0x0000000000000002

julia> 0x1+0x1
0x02
```

Reply:
Right, so there's only `*` left.

For `ByRow(+)`, we should indeed use `reduce(+, ...)` rather than `sum(...)`. So that's different from the fast path for `sum`.